### PR TITLE
Doaa's blog post -> Fix alignment of Arabic text

### PR DESCRIPTION
**Pull request by NPDebs**
:+1::tada: First of all, thanks for taking the time to contribute! :tada::+1:

## FOR CONTRIBUTOR

* [x] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [ ] This PR adds a new blog post
* [x] This PR does something else (explain above)
This PR fixes the alignment of Arabic text in Doaa's welcome blog post.

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [ ] The website builds (a check will fail if not)
* [ ] All images have been added within the Pull Request and they have Alt text
* [ ] If there are paragraphs or text, the key messages are highlighted
* [ ] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [ ] The preview corresponds to the changes described in the Pull Request
* [ ] The code is tidy and passes the linting tests

### CODE REVIEW

File: _posts/2024-11-19-welcoming-our-new-resident-fellow.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ We are thrilled to announce that **[Doaa Mohamed Abdelkader]({% link people.md %
   <div class="column is-3" markdown="1">
   ![Image shows Doaa, an Egyptian woman, wearing a white long-sleeved blouse, a sleeveless mustard dress and a floral hijab. She has her hands cross in front of her, with a broad smile on her face.]({% link images/doaa-portrait.jpg %})
   </div>
-  <div class="column" markdown="1">
+  <div class="column" markdown="1" style="direction: rtl; text-align: right; white-space: pre-line;">>
```
Comment (NPDebs) [inline _posts/2024-11-19-welcoming-our-new-resident-fellow.md:23]:
```suggestion
  <div class="column" markdown="1" style="direction: rtl; text-align: right;">>
```

**Comment by NPDebs** [comment]
@BatoolMM - Following your comment on #923, I have re-aligned the Arabic text. Please take a quick look when you can and let me know if this is good to merge. Thanks again for spotting this!

**Review by BatoolMM — APPROVED** [review]
Now, it's perfect, thank you so much @NPDebs!!

**Comment by NPDebs** [comment]
Ah, now it's perfect like you, Batool. 😍🌹 Thanks a lot!

**Pull request merged**